src/app/components/content-page/content-page.component.html

```diff
@@ -6,17 +6,105 @@
   </nav>
 </header>
 
-<main class="container">
-  content
+<main class="container content centerize">
+  <div class="d-flex software-box-outer">
+    <div class="p-2 card  photo-box-marign">
+      <img class="card-img" [src]="catAvatar3" alt="Cat image">
```

Comment:
Can you get the image on the cards not to stretch?

Reply:
I fixed the stretch on mobile view!